Sources/TuistSigning/SecurityController.swift

```diff
@@ -0,0 +1,63 @@
+import TSCBasic
+import TuistSupport
+
+protocol SecurityControlling {
+    func decodeFile(at path: AbsolutePath) throws -> String
+    func importCertificate(_ certificate: Certificate, keychainPath: AbsolutePath) throws
+    func createKeychain(at path: AbsolutePath, password: String) throws
+    func unlockKeychain(at path: AbsolutePath, password: String) throws
+    func lockKeychain(at path: AbsolutePath, password: String) throws
+}
+
+final class SecurityController: SecurityControlling {
+    func decodeFile(at path: AbsolutePath) throws -> String {
+        try System.shared.capture("/usr/bin/security", "cms", "-D", "-i", path.pathString)
+    }
+
+    func importCertificate(_ certificate: Certificate, keychainPath: AbsolutePath) throws {
+        try importToKeychain(at: certificate.publicKey, keychainPath: keychainPath)
```

Comment:
What about checking first if the certificate already exists before importing it? We should handle that scenario gracefully instead of throwing an error.